var/spack/repos/builtin/packages/kokkos/package.py

```diff
@@ -46,7 +46,27 @@ class Kokkos(Package):
 
     variant('qthreads', default=False, description="enable Qthreads backend")
     variant('cuda', default=False, description="enable Cuda backend")
-    variant('openmp', default=True, description="enable OpenMP backend")
+    variant('openmp', default=False, description="enable OpenMP backend")
+
+    # host architecture variant
+    variant(
+        'host_arch',
+        default='none',
```

Comment:
What if we set the default to be `None` and removed `'none'` from the list of values. Then, later on, we can say something like:
```python
if host_arch_args and not gpu_arch_args:
```

Reply:
Thank you! I made this change and the corresponding one below.